src/components/Footer/Footer.test.js

```diff
@@ -0,0 +1,23 @@
+import Enzyme, {shallow} from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+import Footer from './Footer';
+
+const themeMock = {
+    bgColor: "black",
+    iconFill: "black",
+    iconStroke: "black",
+    textColor: "black",
+    hillColor: "black",
+}
+const socialsMock = [{
+    link: "site",
+    Icon: "Icon",
+    label: "label",
+},]
+
+Enzyme.configure({ adapter: new Adapter() })
+describe('Testing Footer', () => {
+  it('Footer renders without crashing', () => {
+    shallow(<Footer socials={socialsMock} theme={themeMock}/>);
+  });
```

Comment:
Generally, if we are using components with multiple different types of inputs, we want to test a render of different types of states. Most likely it will work but generally this is how conditionally rendered components can be tested without enzyme.

Reply:
in this component all that's being rendered conditionally is the number of icons and links and different inline colors. Should I write some tests to make sure the colors are right and the number of icons rendered are correct?

Reply:
I would just test that each theme renders individually and that should be fine since it's mostly UI.